index.js

```diff
@@ -294,6 +304,12 @@ export default class WebTorrent extends EventEmitter {
     return torrent
   }
 
+  addV2 (torrentId, timeout) {
+    return new Promise((resolve, reject)=> {
```

Comment:
missing space

Reply:
Does this project include an auto formatting setting I've missed or you do the spacing manually?

Reply:
There're no formatter or eslint rules included in package.json. I assume then you have to adjust to the code style manually

Reply:
This would require a lot of effort going forward. Why is this the case?

Reply:
there is, standard which extends eslint

Reply:
I'm unfamiliar with a formatter named "standard" or using eslint as a formatter.
can you link to some instructions about how to set this up?

Reply:
not sure why you're not answering. I can also use my formatter to format only the lines I've touched but I'm not going to do manual style corrections, this is not 1999 we have better things to do with our time.

Reply:
```
$ npx standard --fix index.js 
standard: Use JavaScript Standard Style (https://standardjs.com)
  /home/uri/uriva/webtorrent/index.js:20:35: Parsing error: Unexpected token assert (null)
```

Reply:
Check here: https://standardjs.com/rules

If you haven't setup your code editor to check linting on save, you may run `eslint` on files you've been working on to execute rules which standard provides. I believe it will do the trick. I guess these actions could be automated on MR opening with github actions. Yet I'm not sure

Anyway it's still your responsibility to match your code with the code base you're trying to merge to. Prettier is too opinionated and not an industry standard. I may understand why it's not included in the first place

Reply:
Again, I don't care if it's prettier or not, I care that it's not manual.

Running standard does not work with your code and eslint --fix causes no changes.

Reply:
@uriva I assume this is caused  by you having a different version of `eslint` installed globally vs what is defined in the `package.json`, causing it to possibly not read the `.eslintrc.cjs`.  Does running `npx eslint@8.51.0 .` and `npx eslint@8.51.0 --fix .` while in the root of the project work for you?

Reply:
that works ty!

Reply:
my version was 8.52.0

Reply:
fyi it wasn't even the version, just missing the `.`